### PR TITLE
Update JSON-RPC docs

### PR DESCRIPTION
**Pull request by pyk**
## Motivation

The RPC response:

```
 {
        confirmations: 1,
        difficulty_target: 1844674407370955300,
        hash: '8b897d084c1a60bdcc694bc0d70ebfbe226ac4a578fd63db7dc41074625265ae',
        height: 22418,
        merkle_root: '0aba5eabc3658113651d1d2a1e0fe0b2630b407665f9a22a00372288a1140f91',
        nonce: 100893969,
        pedersen_merkle_root_hash: '424ffec18b3ae156388dd2a5c9b7c447711f10c6fef1dc1bca06da729305f60e',
        previous_block_hash: '120a7b537e968a3aa67e5370f93353c220f3a4d6fa50a82499bd1fd460c994e0',
        proof: '0300000000000000040000000000000007dadbac696b0971d806d6f83f62f57c246984e73d83e1b1f10b6719dc1c946d037c192c0747848134a11f95ff17b9800074129b5396ae486a061f4d32ccf18cffda141be338b8cadd2f345e693ce377e97c7227e3a210cf06a33ed496f1ca9e0000887134aeea7b8b12f9093553c4c645b28f278527823dcb60c15e46adf699383ab0cfdd6d45ef604663a79d6eca66610000002b2d605bb83828ccabd66fba0fc380766070742854333271267d3d220b28ec78093335f00281f26750f5f999bd7f80000300000000000000030e732234254077bded936ba4a62680f612de437e159504b9da3e177677405bc8b02c6748c262eebef814859057c3000002b843b66503ec8cdc8e5fc073eaf245b87388dd0fd11f51e055ecd5c0e3a69293c7f797e76b03b126a4082773170000013445b020cbaa41b1a21808726e38b28041825def211244f6c9da7ea1660bcf02453cd3548703b63ff25ddb649e716b0025a72f90e07f7d78b003d2227832edafe4cdef76734c0f1d7ebe62b2af9dc01d592b88e495255387e326c068888c6d00000200000000000000b0240bfe756ead406b029dc6d2fd19443a09fb80345e2e6e55cc6dea0d1e673a946c249c0d44b022574a3d28da89100001824be56d447dd9aa24356aa1ed755c3cf3932d22b26a3d2b44fb9ae0652c3f72911ece141c4d8b0ff502e6aee2b10f011f286ca12fc0cd0d6234076950ce8f59225cb010a9bcf731db512e827c7722413a2d01e6f685cf432ed1094faceb02010007000000000000001f29274df03523f7f7af8f534bd2c85825155119ee10654923f809195a91b500132841b4b17bbbd498eb7f64800376ad92b867dc3a32645fd0faff8f4f39d108e3995f8b3a316a5fa1e441ab9e546a7d8fded819922797eb4658aaa0a366130e67d2081747d890274be0d38b1be70194b773fbaf224a587da50498e8caa468080b739737e8461a0310bde5dd239e40385c3111dac5b3f4b03e2f3c0646ca9409817de21804ca97f2260516de5c8c3e14c14b1d11c79e7f37c3192932062c7d0eac90f5a12f1b9c4995737402495f6bc2979600002777b1229df5edeb3c2d9a0103000000000000000000000000000000000000000000000000000000000000000200000000000000cbabe6bab0059c97e2a9762afe99d417159ed33d21aec1684d2a20173bca49eee2932957882f99223c41ea88c8d6ef000000b68dac963b095886b23c8bfa26761da3438fc84ecf70d26fea4cceddbec9b6767f78b005a0842e85215503290e5e000135aa9644d3bf75ec1d5e47053b827c1550d89fc818562048483210af30eb530200',
        size: 2627,
        time: 1601128815,
        transactions: [
          '0aba5eabc3658113651d1d2a1e0fe0b2630b407665f9a22a00372288a1140f91'
        ]
      }
```

`pedersen_merkle_root` -> `pedersen_merkle_root_hash`

## Test Plan

(Write your test plan here)

## Related PRs

(Link your related PRs here)

### CODE REVIEW

**Comment by ljedrz** [comment]
Hi @pyk, and thanks for your contribution!

This checks out - the `BlockInfo` object returned by the `get_block` RPC contains a `pedersen_merkle_root_hash` field (as opposed to `pedersen_merkle_root`).

**Review by ljedrz — APPROVED** [review]

File: rpc/documentation/public_endpoints/gettransactioninfo.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Returns information about a transaction from a transaction id.
 | `transaction_proof`     | string | The transaction zero knowledge proof     |
 | `program_commitment`    | string | The program verification key commitment  |
 | `local_data_root`       | string | The local data root                      |
-| `value balance`         | number | The transaction value balance            |
+| `value_balance`         | number | The transaction value balance            |
```
Comment (pyk) [inline rpc/documentation/public_endpoints/gettransactioninfo.md:23]:
```
      {
        digest: 'b058934993bcd2666d45bce96ffb2b96c33a8b7548022d401f2b8fe28366bf0c',
        encrypted_records: [
          '083666c844e972ba41c075a47af332b4f243b820cf99823b7258f7a50fc71c0f01cf17cb031491d9af0718f57759b6720c4aa3753d61f751d73534c8d763c720048e9e065e5546eb59ce41f55a9191bde996eb2387dffc601d9b02aa6d860de30d664807f86a0cacb1c5a4a5433c97ba54313eddb345d999ce1770250c7bc45106206b8b0d86791dc50bdd26e65c57412d249f200e051d822ddb287f29ad8ec10d3f072e1dd6c1cfe6550b3603641a18d01e2b00fad490bf4bca0cb64f00f3e40eb5c57963903e8b1a6f8bfe0688aea5fbe5c0e6a46a9e1e5808c26e358c676d0f3492ae92238c34a4cbe36108422c160b5a7d7b614c91e3b7f4627f2be900c9046900',
          '08deb248498450696128cd91fb416cee828ccbb9bccd8779f743e531cbee3b680654ac3c7a3ad991be6f5719ad9e8952537afa882510f03fb93af33e94230ab20aa7e1930cb136518aee17deb9a4c03df10d03814c94cceabdcf0f9faa6ccb630bf6edaa127ed797b8f833ad95ff8af867b80c0b16b174659d193761b6cb216901badb99f9668b25fb1d87234604c05a38aefbc53c1e3a444d1f4123a64060ab036c24cc0f1d50500d42cc448b5d872d4b83eaf4cc56fb7a962a4827e1a514cf0b3fa8f4452b0b583bd42b143378120bf5d04f506e0215ab5d0b1b89a9ad9d54045cda66cf9c828c45bdde36de5685ba840ee7b00014e4647637e2ce06c0b75c0d0100'
        ],
        local_data_root: 'e637d56259d04cb3c78f8a3c596ea94f33037633df7da13f0de96a7a573e0f08',
        memo: 'fde0cac55344552f8e1cafa26904a2e677b93a1afe23b0de57207a2663e64dea',
        network_id: 1,
        new_commitments: [
          'f0cd40cae2c6606752adde8afcddc415b92281ccc09a0fc9a1bb8a730624ba05',
          'e8dab6b4c6037aab956d24fba489f0abe0efca8c7328f4ad6cb15f18dd569f10'
        ],
        old_serial_numbers: [
          '1ee0444382aea8e0221d197168665fc8185e8b33d76a5185cae411f57855df0f',
          '1dbcddbdc27ecad0093ef202d564d4dd2f72bf97188ae725172743b33c31f810'
        ],
        program_commitment: 'e615d39d2497637649261a732e29c456cae77d3b23f3164ef9bd0e7aae3310f6',
        signatures: [
          '32c590852f3a4767bbd6f3ddddbfaf972dcd5b447c34f5295bcde3ecbb7764016e2a9f437756689f18ae4eea029f00ab329dc25220412a9fa7f1f28319f12c00',
          '2b130c27912c1784d8452686d84d570f0b95b9dbc3ce6e219ed2f36f316f7f04ff86cea5f412e45ada264349bde4350d875140f93022de0544e2cc60d2146c03'
        ],
        size: 1538,
        transaction_metadata: { block_number: 25284 },
        transaction_proof: '56e38aa041cb6ec293a1b6a299e4893c3b37163f5c446c6b86d9258952574775a155d6d58ba9685623d49088fb9fffa99972ce12a7cd398b24e8d6f506af94079139cb2317ac34cfaccd71a72efe19c61ceab6b6cb2ecbac2c9ae719cbc6080078ce0fae0ceef9c02728a0e842c49eb4bb82e956666a7294e5cb276efc97e084e633dddf1c6820825ccb412b527362d9048cb952872ac044006ddec50b1300e2cef523e598aef3b57423b80bdc98b644c3e9bbc3048f1fef03268473fbef94000020fb8c75b4cd0d7e2528c16f610c06e723e3799b2657d34bff2b8cef2a07fd4cc1bec0e224d4b452ed15fd24b356adfb6c2b87385cef8e58a48f40b9ce3b46ddc292d4e8d1ba80995eee08bcd9f3a5d9d56fc9f266a867f4498bad2197f9c0002ea4bd68791c3b7d3a7bc49162ad0024f2dc87c450c4861ca579b49e40a8dce25b5f4e08f4325f4ad735a76d3bc5c7f58bf3d73e0b4eaab3876775ea4135d8d505a0d69b008d2f3ec20663d1d9f7a241a5a702e8ccc8ae7b4c539b5e6f8c370000efcb4bdfb7e3d0f760fd92dbf27b6e7fc4a249e41ca2d758a51839bb66038c7795c025194c5f429daa10d95686229ebd0210b2e9b2bc7bb31ee1636c91152c4ec52fd7c4c76933512df16cf6933243b494c3d82d3255c7abbb1963a63b6b4c003a0cf14e1cac8f3ea501881d4f8380ca3dd6047a503ccd41d21d97e7b340e63efcb23bb7da11a620a5003b05efd4d73487d9666eef15eeb7d534e4ee53ce4a2e190445cf3e6eb70b88276fcd175197279ea7f51db6c69d8ba9b12937a5040e0000',
        txid: '3c462fae82666b7c5c8d84d7e2ca3007c76e20243c6c6af13123fbf1e6902130',
        value_balance: -150000000
      }
```

fix typo:

`value balance` -> `value_balance`

File: rpc/documentation/private_endpoints/createaccount.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Yes
 |   Parameter   |  Type  |         Description         |
 |:------------- |:------:|:--------------------------- |
 | `private_key` | string | An Aleo account private key |
+| `view_key`    | string | An Aleo account view key    |
```
Comment (pyk) [inline rpc/documentation/private_endpoints/createaccount.md:16]: Response from RPC server

```

      {
        address: 'aleo1g89482w35lvyls9k3ekkfn68d0ge78hn6hlrnrrm2j30hnngkcpq5wt6e9',
        private_key: 'APrivateKey1wjJbsPJ2AXiozbgD3UFFzUYDjaXtGbMCwisBZMeLjaUgvQJ',
        view_key: 'AViewKey1qS8oQ3Ep4HqHyatsMpvPj45hn47em57N97TVzaUxmJS5'
      }
```

File: rpc/documentation/public_endpoints/getblock.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Returns information about a block from a block hash.
 | `height`               | number | The block height                                                       |
 | `merkle_root`          | number | The Merkle root of the transactions in the block                       |
 | `nonce`                | number | The nonce for solving the PoSW puzzle                                  |
-| `pedersen_merkle_root` | number | The Merkle root of the transactions in the block using a Pedersen hash |
+| `pedersen_merkle_root_hash` | number | The Merkle root of the transactions in the block using a Pedersen hash |
```
Comment (niklaslong) [inline rpc/documentation/public_endpoints/getblock.md:19]: Nit: table formatting.

**Review by niklaslong — APPROVED** [review]

**Pull request merged**